src/main/java/com/bootcamp/demo/model/User.java

```diff
@@ -4,26 +4,28 @@
 import java.util.UUID;
 
 public final class User {
-    private final UUID userId;
+    private String userId;
     private String email;
     private String firstName;
     private String lastName;
 
+    public User() {}
+
     public User(String email, String firstName, String lastName) {
-        this.userId = UUID.randomUUID();
+        this.userId = UUID.randomUUID().toString().replace("-", "");
```

Comment:
Highlight why you need to replace the '-' in the userId(UUID)? Its not valid UUID then!